test/instructions.cc

```diff
@@ -538,7 +546,8 @@ TEST(Instructions, validate_obvious_maneuver_instructions) {
                      "test/pinpoints/instructions/obvious_maneuver_short_continue.pbf"},
                     expected_routes_size, expected_legs_size, expected_maneuvers_size, maneuver_index,
                     "Take the PA 39 West/Hersheypark Drive exit toward Attractions.",
-                    "Take the exit.", "Take the Pennsylvania 39 West exit.",
+                    "Take the Pennsylvania 39 West, Hersheypark Drive exit toward Attractions.",
```

Comment:
Does not look right - maybe for initial implementation, concentrate on maneuvers that are impacted by long street name - what are your thoughts?